### DISCUSSION TITLE
Share File Html

### DISCUSSION THREAD

**Discussion by shinddayo** [teamalgoritma/community#189] [Q&A] [answered]
Hallo

Saya menyelesaikan capstone project
Dashboard UI Flask.
Setelah dijalankan pythonnya di VS, terbentuk dashboard Html nya sesuai template.
Untuk membukanya bisa dari link yang terbentuk.

Untuk share dashboard yang sudah jadi ke orang lain (dibuka di PC lain atau gadget lain) itu bagaimana kah?
Karena jika dibuka dari link address yang sana,tidak bisa.ataupun jika yang di share hanya file index nya juga tidak seperti basil seharusnya.
Apakah harus dibuat link baru untuk share? Atau harus di generate/di saved dengan cara tertentu kah?

**Comment by yosiaazarya** [comment, marked as answer]
Hai, terkait dengan dashboard agar dapat dimanfaatkan oleh orang lain, maka dapat di deploy ke server (contoh: heroku) sehingga dapat dibaca oleh orang lain. Link yang di generate dari hasil run python script atau flask hanya dapat digunakan secara local. Semoga menjawab! :)

**Reply by shinddayo** [reply]
I see. Saya coba. Terima Masih mas @yosiaazarya
